saas/static/js/djaodjin-saas-vue.js

```diff
@@ -1655,48 +1648,6 @@ Vue.component('user-typeahead', {
     }
 });
 
-// TODO it probably makes sense to use this component in other
-// places where user relations are needed
-Vue.component('user-relation', {
```

Comment:
Why did this component get removed?

Reply:
I grepped and this component is not used anywhere.

Reply:
https://github.com/search?q=org%3Adjaodjin+user-relation&type=Code